tests/air_gapped/test_air_gapped_update.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import os
+from urllib.request import urlretrieve
+
+from lib.host import Host
+
+# These tests are basic tests meant to check the update process in air-gapped environment.
+#
+# Requirements:
+# - a host running XCP-ng 8.3.
+# - with no access to internet
+# - TODO: a snapshot to restore the host to a state without the offline repositories, to integrate when #226
+#   will be merged
+
+REPOS = ['base', 'linstor', 'updates']
+
+
+def air_gapped_download(url: str, host: Host):
+    logging.debug(f"downloading {url}")
+    tmppath = f'/tmp/{os.path.basename(url)}'
+    urlretrieve(url, tmppath)
+    host.scp(tmppath, tmppath)
+    os.remove(tmppath)
+
+
+def test_air_gapped_update(host: Host):
+    # get the required files and install the offline repositories
+    air_gapped_download(
+        'https://raw.githubusercontent.com/xcp-ng/xcp/refs/heads/master/scripts/setup_offline_xcpng_repos', host
```

Comment:
I would create a variable or env var the can override the location, using floating master as default
but I believe a pinned version would make the test more reproductible